config-linux.md

```diff
@@ -594,6 +594,9 @@ The actions, architectures, and operators are strings that match the definitions
 The following parameters can be specified to set up seccomp:
 
 * **`defaultAction`** *(string, REQUIRED)* - the default action for seccomp. Allowed values are the same as `syscalls[].action`.
+* **`defaultErrnoRet`** *(uint, OPTIONAL)* - the errno return code to use.
```

Comment:
It should probably be illegal to specify this for an action that doesn't have an errno value.

Reply:
Thanks, I've added a sentence to clarify it.